### DISCUSSION TITLE
Search models and pick a random one that can speak english

### DISCUSSION THREAD

**Discussion by kevinlinxc** [coqui-ai/TTS#2473] [General Q&A]
I want to randomly choose a speaker that can speak English. From what I can tell (I only read the README), it seems like it's mostly trial and error for figuring this out. Is there actually a way to only see models and speakers that can speak English?

**Comment by erogol** [comment]
[when you do ```tts --list_models```](https://tts.readthedocs.io/en/latest/inference.html)
